src/index.html

```diff
@@ -6,8 +6,42 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article class="product" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="product photo"
+        class="product-photo"
+      >
+
+      <h2 class="product-title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+      <p class="product-code">Product code: 195434</p>
+
+      <div class="container-stars">
+        <div class="stars">
```

Comment:
```suggestion
        <div class="stars stars--4">
```
instead hardcoded variable in scss it is should be modifier in html
and styles should cover all cases of stars amount.
for example if we change modifier on stars--2 should be active two stars without any changes in scss